src/rsa.rs

```diff
@@ -163,8 +165,107 @@ fn parse_public_key<'a>(input: untrusted::Input<'a>) ->
     })
 }
 
+#[allow(non_camel_case_types)]
+enum BIGNUM {}
+#[allow(non_camel_case_types)]
+enum BN_MONT_CTX {}
+
+#[allow(non_camel_case_types)]
+#[repr(C)]
+struct RSA {
```

Comment:
Add a comment that says that this needs to be kept in sync with `rsa_st` in rsa.h. Add a similar comment where `rsa_st` is defined in rsa.h.